add a 'access explorer from dbt cloud features' page (#5643)
fishtown-analytics/docs.getdbt.com: website/docs/docs/collaborate/access-from-dbt-cloud.md

---
title: "Access dbt Explorer from dbt Cloud features"
sidebar_label: "Access from dbt Cloud"
description: "Learn where and how to directly access and interact with dbt Explorer from dbt Cloud features and products."
---

<p style={{ color: '#808080', fontSize: '1.1em' }}>

> Interact with dbt Explorer from other features and products inside dbt Cloud, ensuring you have a seamless experience navigating between resources and lineage in your project. This document guides you on how to access dbt Explorer from the dbt Cloud IDE, jobs, and other product areas.

Author: ```suggestion
Access dbt Explorer from other features and products inside dbt Cloud, ensuring you have a seamless experience navigating between resources and lineage in your project. This document guides you on how to access dbt Explorer from the dbt Cloud IDE, jobs, and other product areas.
```

---
title: "Access dbt Explorer from dbt Cloud features"
sidebar_label: "Access from dbt Cloud"
description: "Learn where and how to directly access and interact with dbt Explorer from dbt Cloud features and products."
---

> <p style={{ color: '#808080', fontSize: '1.1em' }}>

Author: i'm going to add this as a component soon!

> Access dbt Explorer from other features and products inside dbt Cloud, ensuring you have a seamless experience navigating between resources and lineage in your project. 

Reviewer: This color might be a little hard to see for some folks using the dark background

<img width="793" alt="Screenshot 2024-06-11 at 2 10 00 PM" src="https://github.com/dbt-labs/docs.getdbt.com/assets/60105315/0c9be306-6e84-47ab-9006-84ab2e6017d6">

Author: I use dark mode and it looked fine for me! I tested it via accessibility website and it passed but i can check again

Reviewer: Is the point to accentuate this section? Darkening it feels like it puts it in the background rather than highlighting it

Author: It's more like introductory text a for seo. Docs like Microsoft, GitHub and hex do something similar

Reviewer: I guess there's not harm leaving it as is, but as a reader, if I saw that (since we don't adopt that across many pages) my first thought would be "why is that grey?" It feels like something a user may even open an issue for like "hey, this is formatted incorrectly, just wanted to let y'all know"

Author: for sure, it's meant to be like a sub intro and helpful for SEO too. i brought this up to the team [here](https://dbt-labs.slack.com/archives/C04559AB7D2/p1717585832860119) and was going to add this as a component to iterate on it so it automatically changes color for dark and light mode. however, i can bring this up in the docs meeting to further discuss, and am happy to remove it if you feel strongly about this.

Reviewer: I think until we have some consistency across pages with it (instead of just this page) we should hold off on the color at least. If there's a way to implement the SEO optimization and make the color brighter, I'd be all for that. If we're going to implement the darker color, I think it would be better to do it across many pages at once, so this doesn't seem out of place.

</p>

> This guide explains how to access dbt Explorer from various dbt Cloud features, including the dbt Cloud IDE, jobs, and other areas. While the primary way to access dbt Explorer is through the **Explore** link in the navigation, you can also access it from other dbt Cloud features.

Reviewer: Suggest that we retain use of the word `guide` as a descriptor for the /Guides
```suggestion
This page explains how to access dbt Explorer from various dbt Cloud features, including the dbt Cloud IDE and jobs. While the primary way to navigate to dbt Explorer is through the **Explore** link in the navigation, you can also access it from other dbt Cloud features.
```

Author: Good point! I changed it to page?

### dbt Cloud IDE 

> You can enhance your project navigation and editing experience by directly accessing resources from the [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud) to dbt Explorer for model, seed, or snapshot files. This workflow offers a seamless transition between the IDE and the Explorer, allowing you to quickly view and interact with your project's metadata and lineage.

Reviewer: Do we refer to it as "the" Explorer or is it just Explorer?
```suggestion
You can enhance your project navigation and editing experience by directly accessing resources from the [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud) to dbt Explorer for model, seed, or snapshot files. This workflow offers a seamless transition between the IDE and Explorer, allowing you to quickly view and interact with your project's metadata and lineage.
```

Author: I think i missed the 'dbt'!

- **Access dbt Explorer from the IDE** &mdash; In your model, seed, or snapshot file, click the **View in Explorer** icon to the right of your file breadcrumb (under the file name tab). This opens the model, seed, or snapshot file in a new tab, allowing you to view resources/lineage directly in Explorer. 

> - **Use cases** &mdash; Navigate between viewing project metadata and making updates to your models or other resources without switching contexts.

Reviewer: These feel like two different thoughts, rather than bullets in a list. Maybe they could be subsections? Is use cases only going to be one item for the foreseeable future? if so, it could probably be added to the intro paragraph and the access instructions a second paragraph

Author: got it! i was trying to use bullets so its not so text heavy but let me see if i can meet you halfway

> <Lightbox src="/img/docs/collaborate/dbt-explorer/explorer-from-ide.jpg" title="Access dbt Explorer from the IDE by clicking on the 'View in Explorer' icon next to the file breadcrumbs. " />

Author: should i add a rectangle focusing on the actual icon? i feel like i should to help the user directly focus

Reviewer: Yeah, I think adding a rectangle is a good call

### Lineage tab in jobs

> The lineage tab in dbt Cloud jobs displays the lineage associated with the [job run](/docs/deploy/jobs). Access dbt Explorer directly from this tab, allowing you to navigate between both product areas with ease.

Reviewer: Since we're referencing an on-screen element they'll interact with
```suggestion
The **lineage tab** in dbt Cloud jobs displays the lineage associated with the [job run](/docs/deploy/jobs). Access dbt Explorer directly from this tab, allowing you to navigate between both product areas with ease.
```

> - **Access dbt Explorer from the lineage tab** &mdash; From a job, select the lineage tab. Double-click the node in the lineage to open a new tab and view its metadata directly in dbt Explorer.

Reviewer: ```suggestion
- **Access dbt Explorer from the lineage tab** &mdash; From a job, select the **lineage tab**. Double-click the node in the lineage to open a new tab and view its metadata directly in dbt Explorer.
```

> - **Use cases** &mdash; Understand dependencies/relationships of resources in your project.

Reviewer: Same feedback as above regarding mingling the use cases with the instructional material via bulleted lists

> <Lightbox src="/img/docs/collaborate/dbt-explorer/explorer-from-lineage.gif" title="Access dbt Explorer from the lineage tab by double-clicking on the lineage node." />

Reviewer: Nice GIF!!!

### Model timing tab in jobs

> The [model timing tab](/docs/deploy/run-visibility#model-timing) in dbt Cloud jobs displays the composition, order, and time taken by each model in a job run. Access dbt Explorer directly from the modeling timing tab, helping you investigate resources and potentially make changes to improve their performance.

Reviewer: ```suggestion
The [model timing tab](/docs/deploy/run-visibility#model-timing) in dbt Cloud jobs displays the composition, order, and time taken by each model in a job run. Access dbt Explorer directly from the modeling timing tab, which helps you investigate resources and potentially make changes to improve their performance.
```
